ratdb/PMTCHARGE.ratdb

```diff
@@ -19,27 +19,18 @@ valid_end: [0,0],
 charge:[0.,         0.10204082, 0.20408163, 0.30612245, 0.40816327, 0.51020408,0.6122449,  0.71428571, 0.81632653, 0.91836735, 1.02040816, 1.12244898, 1.2244898, 1.32653061, 1.42857143, 1.53061224, 1.63265306, 1.73469388, 1.83673469, 1.93877551, 2.04081633, 2.14285714, 2.24489796, 2.34693878, 2.44897959, 2.55102041, 2.65306122, 2.75510204, 2.85714286, 2.95918367, 3.06122449, 3.16326531, 3.26530612, 3.36734694, 3.46938776, 3.57142857, 3.67346939, 3.7755102,  3.87755102, 3.97959184, 4.08163265, 4.18367347, 4.28571429, 4.3877551,  4.48979592, 4.59183673, 4.69387755, 4.79591837, 4.89795918, 5. ],
 charge_prob:[2.11743463e-02, 3.23358673e-02, 4.80226445e-02, 6.93577003e-02, 9.74160134e-02, 1.33061703e-01, 1.76751397e-01, 2.28328271e-01, 2.86842551e-01, 3.50440685e-01, 4.16363375e-01, 4.81080301e-01, 5.40567191e-01, 5.90702553e-01, 6.27733156e-01, 6.48736514e-01, 6.52001617e-01, 6.37259118e-01, 6.05718026e-01, 5.59901968e-01, 5.03315778e-01, 4.40003520e-01, 3.74075109e-01, 3.09277644e-01, 2.48671091e-01, 1.94441573e-01, 1.47856359e-01, 1.09339717e-01, 7.86326544e-02, 5.49939587e-02, 3.74036587e-02, 2.47400312e-02, 1.59137839e-02, 9.95482716e-03, 6.05593279e-03, 3.58274109e-03, 2.06127938e-03, 1.15330811e-03, 6.27539245e-04, 3.32065295e-04, 1.70880751e-04, 8.55164871e-05, 4.16191893e-05, 1.96981045e-05, 9.06655522e-06, 4.05832890e-06, 1.76660407e-06, 7.47856488e-07, 3.07882044e-07, 1.23264356e-07],
 }
+
+//E.Keener at Penn, updates to charge and probabilities based on darkbox data from Berkeley Jun to Dec 2023.
```

Comment:
In all cases, might be worth including a link to the public website at Berkeley with the data: https://nino.lbl.gov/pmts/